regolith/helpers/a_proposalhelper.py

```diff
@@ -0,0 +1,190 @@
+"""Helper for adding a proposal to the proposals.yml collection.
+"""
+import datetime as dt
+import dateutil.parser as date_parser
+from dateutil.relativedelta import relativedelta
+import sys
+
+from regolith.helpers.basehelper import DbHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import (
+    all_docs_from_collection,
+    get_pi_id,
+)
+
+TARGET_COLL = "proposals"
+
+def subparser(subpi):
+    # Do not delete --database arg
+    subpi.add_argument("--database",
+                       help="The database that will be updated.  Defaults to "
+                            "first database in the regolithrc.json file."
+                       )
+    subpi.add_argument("name", help="A short but unique name for the proposal",
+                       )
+    subpi.add_argument("amount", help="value of award",
+                       )
+    subpi.add_argument("title", help="Actual title of the proposal"
+                       )
+    subpi.add_argument("--begin_date", help="The begin date for the proposed grant "
+                                          "in format YYYY-MM-DD.",
+                       default = 'tbd'
```

Comment:
I think if you are handling this below, as you seem to be, it can be left with no default, which will return ``None`` and then you handle it with ``if not rc.begin_date:`` which makes the intent a bit clearer in the code.

Reply:
That's true - I added defaults everywhere for consistency but they make the code less readable. Will remove them for the `begin_date`, `due_date`, and `duration`